lib/cocoapods-packager/builder.rb

```diff
@@ -49,6 +60,17 @@ def link_embedded_resources
 
     :private
 
+    def build_dynamic_framework(platform, defines, output)
+      UI.puts 'Building dynamic Framework'
+
+      clean_directory_for_dynamic_build
+      if platform.name == :ios
+        build_dynamic_framework_for_ios(platform, defines, output)
+      else
+        build_dynamic_framework_for_mac(platform, defines, output)
```

Comment:
Does the packager even support watchOS and tvOS at this point? Looking through the code, I don't see an mention of `-sdk watchos`. If I omit an explicit SDK, (ie `" -sdk iphoneos"`) does it build for all architectures. 

Also, if you look at the impl of `build_dynamic_framework_for_mac`, there is only a comment, not an implementation...as I'm not entirely sure what is needed there. If an implementation for that is required to get this PR merged, I'm happy to look into it.